packages/rest/src/explorer.component.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/rest
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Component} from '@loopback/core';
+
+export class ExplorerComponent implements Component {
+  staticAssets = [
+    {
+      path: '/explorer1',
+      rootDir: './public',
```

Comment:
How is this relative path going to be resolved into an absolute one? IMO, it should be the responsibility of the component to resolve the path. E.g. `rootDir: path.resolve(__dirname, '../../public')`